### PR TITLE
num tables and preface title update

### PR DESCRIPTION
**Pull request by brittany-johnson**
Remove summary attribute from numbered tables, add option to pass title element to `BakePreface`

### CODE REVIEW

File: lib/kitchen/directions/bake_numbered_table/v1.rb [hunk 1]
```diff
@@ -4,6 +4,7 @@ module Kitchen::Directions::BakeNumberedTable
   class V1
 
     def bake(table:, number:, always_caption: false)
+      table.remove_attribute('summary')
```
Comment (brittany-johnson) [inline lib/kitchen/directions/bake_numbered_table/v1.rb:7]: [in this PR](https://github.com/openstax/kitchen/pull/249) I thought I had removed the summary attr BUT it was still there but with different data

File: lib/kitchen/directions/bake_preface/main.rb [hunk 2]
```diff
@@ -3,8 +3,8 @@
 module Kitchen
   module Directions
     module BakePreface
-      def self.v1(book:)
-        V1.new.bake(book: book)
+      def self.v1(book:, title_element:)
```
Comment (brittany-johnson) [inline lib/kitchen/directions/bake_preface/main.rb:6]: add an option because am gov is a div instead of h1

Comment (LostMojo) [reply lib/kitchen/directions/bake_preface/main.rb:6]: is there a reason to define them seperately rather than giving the second argument a default?

**Review by LostMojo — COMMENTED** [review]
Loos good! I only have one question about defining two v1 methods-- I've been doing it by giving the second argument a default value, but is there a benefit to defining it twice like this?

**Review by LostMojo — APPROVED** [review]

**Pull request merged**